packages/vhd-lib/merge.js

```
@@ -256,11 +256,14 @@ class Merger {
     // mergeTargetChild is already in an incomplete state, its blocks have been transferred to parent
     let oldTarget
     if (isVhdAlias(mergeTargetChild)) {
-      oldTarget = resolveVhdAlias(handler, mergeTargetChild)
+      oldTarget = await resolveVhdAlias(handler, mergeTargetChild)
     }
 
     try {
       await handler.rename(parent, mergeTargetChild)
+      if (oldTarget !== undefined) {
+        await VhdAbstract.unlink(handler, oldTarget)
+      }
     } catch (error) {
       // maybe the renaming was already successfull during merge
       if (error.code === 'ENOENT' && this.#isResuming) {
```

What if it's another error?

Is the error simply swallowed?